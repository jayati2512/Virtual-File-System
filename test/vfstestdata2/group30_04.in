createvfs demo 40
mountvfs demo
makedir /institute/college/iiit/imtech/comp.sci/python marks
makedir /institute/college/iiit/imtech embedded_systems
makedir /institute/college/iiit/imtech/comp.sci/python faculty
addfile /institute/college/iiit/imtech/comp.sci cse /usr/include/stdio.h
movedir /institute/college/iiit/imtech/comp.sci/python/faculty /institute/college/iiit/imtech/comp.sci/cse
addfile /institute/college/iiit/imtech/comp.sci/cse junkfile /usr/include/stdlib.h
listdir /institute/college 1 /tmp/listout4
unmountvfs demo
