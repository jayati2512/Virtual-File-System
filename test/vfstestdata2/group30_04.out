createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
addfile_FAILED FILE_TOO_LARGE
movedir_FAILED DESTINATION_CANNOT_BE_FILE
addfile_FAILED CANNOT_WRITE_TO_DATAFILE
listdir_SUCCESS
unmountvfs_SUCCESS
